src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin flex {
```

Comment:
it's really nice to see you've used mixin. 
I'd recommend you to update a little, so you will be more flexible;

```
@mixin flex($type: flex, $align: false, $justify: false, $direction: false) {
  display: $type;
  @if ($align) {
    align-items: $align;
  }
  @if ($justify) {
    justify-content: $justify;
  }
  @if ($direction) {
    flex-direction: $direction;
  }
}
```

Usage:
```
@include flex($align: center);
@include flex($align: center, $justify: center);
@include flex($direction: column, $justify: flex-end, $align: center);
```
and so on